scheduler/docs/configuration.adoc

```diff
@@ -266,6 +266,42 @@ Optionally, you can include a "rebalancer" stanza.  If you do, on startup, Cook
   To obtain useful DRU metrics in this situation, you can either adjust your share
   settings (recommended), or increase the dru-scale setting to e.g. 10^300.
 
+[[optimizer]]
+====Optimizer configuration
+
+The link:optimizer.md[optimizer] is a component that can make more global decisions about the cluster, job placement and autoscaling.
+By default, Cook will use a no-op optimizer. To plug in an implementation, add the optional "optimizer" stanza. The three pluggable pieces,
+host-feed, optimizer and schedule-consumer, each have the same structure to configure. A :create-fn:: key which is a namespaced symbol
+on the class path and a :config:: key which is an arbitrary map. See the example below.
+
+ :host-feed::
+    The implementation for the host feed to use. The host feed returns a list of maps where each map describes a host type that can be purchased.
+
+ :optimizer::
+    The implementation for the optimizer to use. The optimizer accepts the output of the host feed, the queue, the running tasks and the available offers and outputs a schedule of suggestions.
+
+ :schedule-consumer::
```

Comment:
Hasn't the `consumer` support been removed?

Reply:
yea, good catch, forgot to grep in docs